web/models/release.ex

```diff
@@ -10,7 +10,7 @@ defmodule HexWeb.Release do
     timestamps
 
     belongs_to :package, Package
-    has_many :requirements, Requirement
+    has_many :requirements, Requirement, on_replace: :delete
```

Comment:
I needed it for updating a release, when requirements changed (existing behaviour). See this test: https://github.com/wojtekmach/hex_web/blob/d029fc8d2f5ae11bda1188c3b1fda12e030ddc48/test/models/release_test.exs#L105.